stubs/openpyxl/openpyxl/worksheet/worksheet.pyi

```diff
@@ -24,106 +31,106 @@ class Worksheet(_WorkbookChild):
     @property
-    def columns(self): ...
-    def set_printer_settings(self, paper_size, orientation) -> None: ...
-    def add_data_validation(self, data_validation) -> None: ...
-    def add_chart(self, chart, anchor: Incomplete | None = ...) -> None: ...
-    def add_image(self, img, anchor: Incomplete | None = ...) -> None: ...
-    def add_table(self, table) -> None: ...
+    def columns(self) -> Generator[Iterable[Cell], None, None]: ...
+    def set_printer_settings(self, paper_size: Any, orientation: Any) -> None: ...
+    def add_data_validation(self, data_validation: Any) -> None: ...
+    def add_chart(self, chart: Any, anchor: Incomplete | None = ...) -> None: ...
+    def add_image(self, img: Any, anchor: Incomplete | None = ...) -> None: ...
+    def add_table(self, table: Table) -> None: ...
     @property
-    def tables(self): ...
-    def add_pivot(self, pivot) -> None: ...
+    def tables(self) -> TableList: ...
+    def add_pivot(self, pivot: Any) -> None: ...
     def merge_cells(
         self,
-        range_string: Incomplete | None = ...,
-        start_row: Incomplete | None = ...,
-        start_column: Incomplete | None = ...,
-        end_row: Incomplete | None = ...,
-        end_column: Incomplete | None = ...,
+        range_string: str | None = ...,
+        start_row: int | None = ...,
+        start_column: int | None = ...,
+        end_row: int | None = ...,
+        end_column: int | None = ...,
     ) -> None: ...
     @property
-    def merged_cell_ranges(self): ...
+    def merged_cell_ranges(self) -> list[CellRange]: ...
     def unmerge_cells(
         self,
-        range_string: Incomplete | None = ...,
-        start_row: Incomplete | None = ...,
-        start_column: Incomplete | None = ...,
-        end_row: Incomplete | None = ...,
-        end_column: Incomplete | None = ...,
+        range_string: str | None = ...,
+        start_row: int | None = ...,
+        start_column: int | None = ...,
+        end_row: int | None = ...,
+        end_column: int | None = ...,
     ) -> None: ...
-    def append(self, iterable) -> None: ...
-    def insert_rows(self, idx, amount: int = ...) -> None: ...
-    def insert_cols(self, idx, amount: int = ...) -> None: ...
-    def delete_rows(self, idx, amount: int = ...) -> None: ...
-    def delete_cols(self, idx, amount: int = ...) -> None: ...
-    def move_range(self, cell_range, rows: int = ..., cols: int = ..., translate: bool = ...) -> None: ...
-    @property
-    def print_title_rows(self): ...
+    def append(self, iterable: Iterable[Any]) -> None: ...
+    def insert_rows(self, idx: int, amount: int = ...) -> None: ...
+    def insert_cols(self, idx: int, amount: int = ...) -> None: ...
+    def delete_rows(self, idx: int, amount: int = ...) -> None: ...
+    def delete_cols(self, idx: int, amount: int = ...) -> None: ...
+    def move_range(self, cell_range: CellRange, rows: int = ..., cols: int = ..., translate: bool = ...) -> None: ...
```

Comment:
`cell_range` can also be a `str`: https://foss.heptapod.net/openpyxl/openpyxl/-/blob/branch/3.0/openpyxl/worksheet/worksheet.py#L771-772

```suggestion
    def move_range(self, cell_range: CellRange | str, rows: int = ..., cols: int = ..., translate: bool = ...) -> None: ...
```